backend/services/AccountService.ts

```diff
@@ -79,15 +78,15 @@ export class AccountService {
    * @returns A Promise that resolves to an AuthAccountResponse object containing the account information.
    * @throws If there was an error retrieving the account information.
    */
-  public async getAccount(msaId: string): Promise<AuthAccountResponse> {
+  public async getAccount(msaId: string): Promise<Partial<AuthAccountResponse>> {
     try {
       const response = await this.client.AccountsControllerV1_getAccount(msaId);
       logger.debug(
         `AccountService: getAccount: Got account for msaID:(${msaId}), data:(${JSON.stringify(response.data)})`
       );
       return {
         msaId: msaId,
-        handle: response.data.handle,
+        handle: response.data.handle as { base_handle: string; canonical_base: string; suffix: number } | undefined,
```

Comment:
Note: created issue to have this handle type defined in the OpenAPI spec from account-service
https://github.com/AmplicaLabs/gateway/issues/303